plugins/search/api-report.md

```diff
@@ -65,10 +66,16 @@ export type FiltersState = {
 //
 // @public (undocumented)
 export const HomePageSearchBar: ({
-  placeholder,
-}: {
-  placeholder?: string | undefined;
-}) => JSX.Element;
+  className: defaultClassName,
+  ...props
+}: Partial<Omit<SearchBarBaseProps, 'onChange' | 'onSubmit'>>) => JSX.Element;
+
+// Warning: (ae-missing-release-tag) "HomePageSearchBarProps" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
I think it's good to start to get the ball rolling on habitually adding `@public` and a proper docstring on things that go into the api report, so that these files eventually end up having _no_ warnings or `(undocumented)` anywhere in them. Not just for this file's sake, but since this actually is part of the public surface of our packages that people rely on, and because it'll more and more start to go into our auto generated API docs on the microsite.